benchmarks/user/make-azure-jobs.py

```diff
@@ -0,0 +1,80 @@
+import os
+
+import click
+
+from benchmark import option_simulation
+import devito
+
+
+@click.group()
+def menu():
+    pass
+
+
+@menu.command(name='generate')
+@option_simulation
+@click.option('-id', default='bench', help='Job ID')
+@click.option('-image', default='mloubout/devito-bench:v0.1', help='Devito docker image')
+@click.option('-arch', default='gcc-8', help='Devito architecture')
+@click.option('-nt', default='4', help='OpenMP number of threads')
+@click.option('-runtime', default='mpich', help='MPI runtime')
+@click.option('-ppn', default='1', help='Processes per node')
+@click.option('-tn', default=50, help='Number of timesteps')
+def generate(**kwargs):
+    join = lambda l: ' '.join('%d' % i for i in l)
+    args = dict(kwargs)
+    args['shape'] = join(args['shape'])
+    args['space_order'] = join(args['space_order'])
+
+    args['home'] = os.path.dirname(os.path.dirname(devito.__file__))
+
+    template_header = """\
+job_specifications:
+- id: %(id)s
+  shared_data_volumes:
+  - azureblob_vol
+  tasks:
+  - docker_image: %(image)s
+    environment_variables:
+      DEVITO_ARCH: %(arch)s
+      OMP_NUM_THREADS: %(nt)s
+      OMP_PROC_BIND: 'close'
+      DEVITO_MPI: 1
+      DEVITO_OPENMP: 1
+      OMP_PLACES: 'cores'
+      DEVITO_LOGGING: 'DEBUG'
+      LC_ALL: 'C.UTF-8'
+      LANG: 'C.UTF-8'
+    default_working_dir: container
+    multi_instance:
+      num_instances: pool_current_dedicated
```

Comment:
this can also be fixed to a number, say you start a pool with 8 nodes you can decide to use only half or something
```
multi_instance:
       num_instances: pool_current_dedicated/2
      mpi:
         runtime: mpich
         processes_per_node: 1
         options:
           - --bind-to socket
```

For example will use half the nodes with one worker per node